File: Backend/app/src/main/java/com/example/demo/dto/RegisterResponse.java
package com.example.demo.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDate;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class RegisterResponse {
    private Long id;
    private String fullName;
    private String username;
    private String email;
    private String phoneNumber;
    private String img_url;
    private String gender;
    private String date_of_birth;
}

File: Mobile/app/src/main/java/com/example/myapplication/data/dto/RegisterResponse.kt
package com.example.myapplication.data.dto

import java.time.LocalDate

data class RegisterResponse(
    val id: Long?,
    val fullname: String?,
    val username: String?,
    val email: String?,
    val phoneNumber: String?,
    val img_url: String?,
    val gender: String?,
    val date_of_birth: String?
)